backend/db/mongo_test.go

```diff
@@ -0,0 +1,82 @@
+package db
```

Comment:
I moved all mongo tests to database_tests.go, did you pull main branch into this branch?

Reply:
I did but I was just looking into the same thing... I'll reconcile those and re-commit